libphx/src/Window.cpp

```diff
@@ -6,11 +6,15 @@
 #include "Viewport.h"
 #include "Window.h"
 #include "WindowMode.h"
+#include "Tex2D.h"
+#include "Resource.h"
+extern uchar* Tex2D_LoadRaw(cstr path, int* sx, int* sy, int* components);
```

Comment:
I am not completely sure what you want here? I don't seem to have an INTERNAL_API defined.
But do you mean to move the function into Tex2D as a static?
It is currently in its own file Tex2D_load.cpp hence the externs scattered around to reference it, yes I copied what was already there but happy to tidy up if I can determine what you are asking for.

Reply:
Sorry, I meant `PRIVATE`, not `INTERNAL_API`, I edited my comment after posting.

I understand the implementation is in Tex2D_Load.cpp, however we should be declarating the function interface in the Tex2D.h header file. I didn't realise that Tex2D_LoadRaw was being declared in multiple places like TexCube.cpp, however all of those should get consolidated into Tex2D.h, rather than duplicating the function prototype across the places that use Tex2D_LoadRaw.

I think what would be best is:
* Remove `extern` as that keyword is redundant for functions
* Delete the Tex2D_LoadRaw declarations from Tex2D.cpp and TexCube.cpp, and add it to Tex2D.h (which will make it available in those cpp files).
* Add `PRIVATE` in place of `extern`, which makes sure the symbol is not exported on some platforms.

Reply:
Done. And gave same treatment to Tex2D_Save_Png().